README.md

```diff
@@ -99,20 +80,25 @@ the following shell command:
 nix run github:pjones/plasma-manager
 ```
 
-## Contributions and Maintenance
+## Getting started
```

Comment:
Have you considered moving the Getting Started section first? Or maybe below the list of what we can do with plasma-manager, but before sections talking about the configuration itself (which needs some basic code infrastructure given by the Getting Started section)

Reply:
I agree, I think that makes more sense